Changes

```diff
@@ -47,6 +47,9 @@ be mentioned in the 4.06 section below instead of here.)
 ### Internal/compiler-libs changes:
 
 ### Bug fixes
+- MPR#7661: fix faulty compilation of patterns of extension types
+  (Luc Maranget, review by Thomas Refis and Gabriel Scherer, report by
+  Abdelraouf Ouadjaout and Thibault Suzanne)
```

Comment:
IIRC we want to keep things ordered by MPR/GPR number in the changelog, so this is misplaced.

Reply:
Yes, and the entry should also reference the GPR: `MPR#7661, GPR#1459: ...`